lib/msf/core/payload.rb

```diff
@@ -295,7 +295,7 @@ def handler_klass
   # Returns the session class that is associated with this payload and will
   # be used to create a session as necessary.
   #
-  def session
+  def session_klass
```

Comment:
Potential damage is we root out more bugs. I'm not saying that's bad; we found a ton of bugs as a result of datastore normalization changes too. Definitely for fixing fundamental problems, and dealing with the fallout as best we can.